rpa-builder/6.6/modules/ROOT/pages/toolbox-flow-control-force-error-state.adoc

```diff
@@ -1,8 +1,22 @@
+= Force Error State
 
+Forces the _Workflow_ into an error state and writes the error message to the `Error Out` workflow variable.
 
-= Force Error State
+If you don't handle this error within a _Managed Block_, the _Workflow_ aborts at this point and returns with

```

Comment:
```suggestion
If you don't handle this error within a _Managed Block_, the _Workflow_ aborts the request at this point and returns with

```
Or maybe the process? But I know RPA uses these same sentences somewhere else. So you can discard the comment.